test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo14-addon-knowledge @git+https://github.com/dzungtran89/knowledge@14.0-mig-knowledge#subdirectory=setup/knowledge
```

Comment:
I don't think this can be merged like this... If you are referencing this just for testing, please wait for the other PR to be merged and remove this reference...

Reply:
indeed, the dependencies should be merged first